Traversal.fs

```diff
@@ -776,6 +776,17 @@ let pushVar (ctx : TraversalContext<'Var>) (v : CTyped<'Var>)
     | c -> fail (ContextMismatch ("vars context", stripVars c))
 
 /// <summary>
+///     As <see cref="pushVar"/>, but does not crash on invalid context.
```

Comment:
This was borne out of frustration mainly, and just means I can get away with not doing a lot of traversal mangling to add and remove contexts just so `pushVar` works.